worksheet_A/TerminalHacking.cpp

```diff
@@ -7,8 +7,22 @@
 const int wordLength = 5;
 const int numberOfWords = 15;
 
+int getLikeness(std::string secretWord, std::string playersWord)
+{
+	int Likeness = 0;
```

Comment:
Follow a consistent naming style -- all of your other variable names begin with a lowercase letter